flask_lambda.py

```diff
@@ -90,6 +91,8 @@ def start_response(self, status, response_headers, exc_info=None):
 
 class FlaskLambda(Flask):
     def __call__(self, event, context):
+        if 'isBase64Encoded' in event and event['isBase64Encoded']:
```

Comment:
Don’t check if a key is in the duct and then fetch it. Use `.get` instead.

Reply:
For some reason I thought .get was deprecated, will update..

Reply:
Updated